web/forms.py

```
@@ -1720,25 +1720,29 @@ class Meta:
             "max_attempts",
+            "passing_score",
+            "ai_auto_correction",
+            "enable_copy_past_and_text_selection",
         ]
         widgets = {
             "title": TailwindInput(attrs={"placeholder": "Quiz Title"}),
             "description": TailwindTextarea(attrs={"rows": 3, "placeholder": "Quiz Description"}),
-            "subject": TailwindSelect(),
             "status": TailwindSelect(),
             "time_limit": TailwindNumberInput(
                 attrs={"min": "0", "placeholder": "Time limit in minutes (leave empty for no limit)"}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)